deploy/helm/sumologic/values.yaml

```diff
@@ -218,6 +218,68 @@ sumologic:
       ## Set provider service (either fluentd or otelcol).
       provider: fluentd
 
+    collector:
+      enabled: false
```

Comment:
Can we change the name of this key to make it obvious it is about otelcol?
Current name would be OK in v4 when we'll have only single agent type but right now I think we should be more explicit.

I suggest make it:
```
sumologic:
  logs:
    collector:
      otelcol:
        enabled: true
```

Reply:
Done.